pkg/core/transactor/listener.go

```diff
@@ -206,6 +228,17 @@ func (t *Transactor) handleSendStandardTx(req *node.TransferRequest) (*node.Tran
 		return nil, errWalletNotLoaded
 	}
 
+	// Are we allowed to send a transaction?
+	// TODO: use a parent context
+	progress, err := t.getSyncProgress()
```

Comment:
Should we disable Transfer transaction too? I thought we need this only for Bid and Stake transactions.

In case your wallet has balance and recent blocks do not update your wallet, you should be allowed to submit at least Transfer, I think.

Reply:
Yes. You can't guarantee that any type of transaction you make before being synced will be correct. So to avoid potentially polluting the network with an invalid transaction, we restrict it on every type.